### PR TITLE
Feature/slider draft

### PR DESCRIPTION
**Pull request by kjec2f**
**Describe pull-request**  
_Describe what the pull-request is about_
Cleaned and re-organized the slider component
**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #

**How to test**  
_Add description how to test if possible_
1. Go to storybook -> slider

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._

### CODE REVIEW

**Comment by alexlioliopoulos** [comment]
Enabled/Disabled Track height
Starting point of the slider circle

Just leaving them here as comments for further improvement of the slider component in the future. No need for you Kewser to work on them.

**Comment by mmexvr** [comment]
There are some stuff that needs to be cleaned up and removed. I think we simplify it a bit also. especially the basic components.

File: components/src/components/slider/readme.md [hunk 1]
```diff
@@ -7,13 +7,12 @@
...
-| `max`      | `max`       |             | `string` | `'100'`     |
-| `min`      | `min`       |             | `string` | `'0'`       |
-| `type`     | `type`      |             | `string` | `undefined` |
-| `value`    | `value`     |             | `string` | `'20'`      |
-| `valueTwo` | `value-two` |             | `string` | `'70'`      |
+| Property | Attribute | Description            | Type     | Default     |
+| -------- | --------- | ---------------------- | -------- | ----------- |
+| `max`    | `max`     | Maximum value of input | `string` | `'100'`     |
+| `min`    | `min`     | Minmum value of input  | `string` | `'0'`       |
+| `type`   | `type`    | type of slider         | `string` | `undefined` |
```
Comment (mmexvr) [inline components/src/components/slider/readme.md:14]: Fix the type attribute so it is not undefined on default

File: demo/angular/package-lock.json [hunk 2]
```diff
@@ -1,17816 +1,8 @@
 {
   "name": "sdds-angular-demo",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```
Comment (mmexvr) [inline demo/angular/package-lock.json:4]: No need to commit

**Comment by mmexvr** [comment]
Lets wait until next release, some of the fixes is needed. I will take over

**Review by theJohnnyMe — APPROVED** [review]
Code needs detailed check and simplification. I am approving it so it can be merged with our current code. I am putting it under pre-alpha status in Storybook in case we want to come back to it sometimes.

**Pull request merged**